spec/sidekiq/jobs/inning_rollover_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+require 'sidekiq/testing'
+Sidekiq::Testing.inline!
+
+RSpec.describe InningRolloverJob, type: :job do
+  before :each do
+    Sidekiq::Testing.fake!
+    @inning1 = create(:inning, :is_current)
+    @inning2 = create(:inning, :not_current_future)
+  end
+
+  describe 'queueing' do
+    it 'queues the job' do
+      expect { InningRolloverJob.perform_async(@inning2.id) }.to change(InningRolloverJob.jobs, :size).by(1)
+      Sidekiq::Worker.drain_all
```

Comment:
Here's an instance where you probably do want the testing mode to be `fake` so that it actually queues the job.